### DISCUSSION TITLE
Error when launching the jar - Injected field not provided

### DISCUSSION THREAD

**Discussion by tnolden** [eclipse-edc/Connector#3134] [Q&A] [answered]
I have created a connector with the current build file from the documentation "build-your-own-connector" with the version: 0.0.1-milestone-9. The build was successful, but when trying to start the jar I get the following error:

SEVERE 2023-06-01T06:38:22.2507255 Error booting runtime: The following injected fields were not provided: Field "dataPlaneInstanceStore" of type [interface org.eclipse.edc.connector.dataplane.selector.spi.store.DataPlaneInstanceStore] required by org.eclipse.edc.connector.catalog.CatalogDefaultServicesExtension
org.eclipse.edc.spi.system.injection.EdcInjectionException: The following injected fields were not provided:
Field "dataPlaneInstanceStore" of type [interface org.eclipse.edc.connector.dataplane.selector.spi.store.DataPlaneInstanceStore] required by org.eclipse.edc.connector.catalog.CatalogDefaultServicesExtension
        at org.eclipse.edc.boot.system.DependencyGraph.of(DependencyGraph.java:102)
        at org.eclipse.edc.boot.system.ExtensionLoader.loadServiceExtensions(ExtensionLoader.java:119)
        at org.eclipse.edc.boot.system.runtime.BaseRuntime.createExtensions(BaseRuntime.java:151)
        at org.eclipse.edc.boot.system.runtime.BaseRuntime.boot(BaseRuntime.java:201)
        at org.eclipse.edc.boot.system.runtime.BaseRuntime.boot(BaseRuntime.java:84)
        at org.eclipse.edc.boot.system.runtime.BaseRuntime.main(BaseRuntime.java:72)

I hope someone can help me.

**Comment by paullatzelsperger** [comment, marked as answer]
the `NoopPrivateKeyResolver` doesn't exist anymore. Please make sure you have purged all Gradle (`~/.gradle/...`) and Maven (`~/.m2`) caches and are using the latest version of EDC (`0.1.0` at the time of writing)

**Reply by tnolden** [reply]
I cleared the cache and I'm using the version "0.1.0" now. . After rebuilding the hole Repository the error is gone. Thanks!

**Reply by Silhim** [reply]
I have the exact same issue as tnolden. But when I use edcVersion = "0.1.0" in the build.gradle.kts file I get errors in the build and that the maven repos don't exist. How would you fix that?
